### DISCUSSION TITLE
how do yo make your stylesheet prior than the admin one

### DISCUSSION THREAD

**Discussion by ghost** [fabiocaccamo/django-admin-interface#281] [Q&A]
I'm trying to make something similar but with just static files and template files to extend the admin base.html and base_site.html.

When I install your package the admin colors change straight to green and I'm wondering how you make your green style prior to the blue one from the admin.

Thanks,

**Comment by fabiocaccamo** [comment]
@debbieacciona your app must be listed in `settings.INSTALLED_APPS` before `django.contrib.admin`.

**Reply by ghost** [reply]
hi, yes! i did that but my link stylesheet is reflected into head but is not overwritting the admin base.html and admin base.css
any help
<img width="573" alt="Captura de pantalla 2023-05-08 a las 10 06 15" src="https://user-images.githubusercontent.com/112860564/236794540-c324f43b-aec4-4864-ac34-d06bac532d10.png">

**Reply by fabiocaccamo** [reply]
If you want to override the admin css, it must have the same path/filename, in this case your file should be named `base.css`

**Reply by ghost** [reply]
i've changed it but still not working.

I have this structure:

MyPackage/
    __ init __.py
    setup.py
    static/
        admin/
            css/
                base.css
            img/
                *.svg
    templates/
        admin/
            base.html
            
the picture above is exactly the same with this structure and changing admin.css by base.css

any idea?

**Reply by fabiocaccamo** [reply]
Have you run `python manage.py collectstatic` after having renamed the file?

**Reply by ghost** [reply]
into the project or into the package i want to do?

**Reply by fabiocaccamo** [reply]
In the virtualenv where you are testing your project / package.

**Reply by ghost** [reply]
yes i did and it's working ^^Many thanks
